app/components/dashboard/in_progress_row_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="col-4">
-    <%= show_collection_link %> &gt; <%= work_link %> <%= edit_work_link %>
+    <%= show_collection_link %> &gt; <%= work_link %> <%= helpers.turbo_frame_tag dom_id(work, :edit), src: edit_button_work_path(work), target: '_top' %>
```

Comment:
this is an edit link on the dashboard, switching to use turbo like other edit work links on the dashboard (so that only edit links on the work detail page use the edit link component)